doc/user/COMMANDS.md

```diff
@@ -0,0 +1,78 @@
+# Commands
+
+F3D will provides access to commands through interactions configuration and command line widget in the future.
+Commands let you trigger specific behavior that may not be available otherwise.
+Please note commands are currently experimental and the behavior, names and arguments may change without deprecation.
+
+Standard commands have the following syntax:
+
+`action libf3d.option.name [values]`
+
+Specific commands have the following syntax:
+
+`specific_action [arg] [values]`
```

Comment:
I still disagree about differentiation of standard/specific commands.  
For me, it should be documented as `action [args]`.  
Then you can explain below that a few commands are related to options state modification (set/toggle/unset/reset/print) in which case the first argument correspond to the option name.

Reply:
Some specific commands are also related to option state modification, so that not really the right distinction

Reply:
Ok, I agree it may not be the right distinction but I still think there is no reason to document a difference between the standard and specific commands.

Reply:
The value I see is for documentation and ease of use for user. They would know that using a standard action require an option name.

Reply:
And that should be documented at such in commands documentation. I just disagree about the specific command category.